### ISSUE TITLE
Status Bar - iPhone 12 Max Pro

### ISSUE THREAD

**Issue by cvcoder1** [SwiftKickMobile/SwiftMessages#477]
Using .cardView, precisely as in the example provided on the README, the banner makes the status bar disappear (if the banner is appearing while the application is launching). 

I am using the MessageView to display internet connection / disconnection with firebase. 

I am testing and accidentally connected to the public WiFi at my apartment complex; which I wasn't configured to (testing WiFi and no WiFi scenarios). And discovered this issue, where if the card displays as the app is launching or right after the app launches, the status bar becomes invisible, it doesn't go back to normal until you leave the view controller you launched the app on.

**Comment by cvcoder1** [comment]
Note: I put iPhone 12 Max Pro because I tested this exact occurrence dozens of times on iPhone 11 Max Pro and it doesn't occur on that device. Note: I am testing on real device, and not on simulators.

**Comment by cvcoder1** [comment]
Update: I am overriding the app's user interface system. Moreover,this issue only occurs when the device is in dark mode, and our app is in light. How can we fix this and can you please ensure the card view properly handles all theme based scenarios? Thanks.

**Comment by wtmoose** [comment]
Can you attach a sample app that reproduces the issue?

**Comment by cvcoder1** [comment]
Here you go: https://user-images.githubusercontent.com/85153827/128804729-81516a36-2afc-4df9-a23f-876a7d6157d1.MOV


First, status bar is proper color.

Then, I swipe up to control center and turn airplane mode on. 

SwiftMessages displays the banner, and the status bar display changes to .light as opposed to .dark as it should be, according to the userInterfaceStyle. 

Note: This is a tab bar app. 

Next, I tap the profile tab button and it switches the ui navigation controller to a new root view controller, and the status bar regulates back. 

It just seems that the status bar style is being overridden and defaulted to the device's state.

**Comment by cvcoder1** [comment]
On third thoughts, I don't think this has anything to do with the device model, but the fact that I had one device's system appearance settings set to automatic, and this was late at night. Therefore, my iPhone 11 Max Pro was in light mode 24/7 and my iPhone 12 Max Pro was in automatic. 👍🏻

Seems like it's just an issue with the status bar style? @wtmoose

**Comment by cvcoder1** [comment]
And, I was right: tested. ![image](https://user-images.githubusercontent.com/85153827/128805270-7c339b68-d1b1-497e-a076-8f6fa8fa7338.jpeg)

**Comment by cvcoder1** [comment]
Also I want to mention this does not only happen at launch. App launch is actually irrelevant aside from the way I firstly discovered the issue..I believe it's 100% related to the status bar style just defaulting or something. 

Let me know if you need anything else. We really appreciate your library and have recommended it to several of our colleagues. In addition, my company plans to sponsor ~ $20/month starting later this month. 👍🏻

**Comment by cvcoder1** [comment]
> Can you attach a sample app that reproduces the issue?

Could you please provide an update on the status of this? Should I use something other than .cardView? Is there a hack around or will this be fixed in the next update? Thanks!

**Comment by wtmoose** [comment]
The status is, I’m on vacation this week :)

**Comment by cvcoder1** [comment]
Sounds great :) hope it went well. Do you think you could have a new version with this fixed, prior to September 1st? Or could you please recommend any interim solutions in the mean time? Thank you very much, Timothy.

**Comment by wtmoose** [comment]
You haven't supplied me with the sample project I asked for yet. However, I went ahead and created my own, but haven't been able to reproduce your issue.

For next steps, I need you to take the project below and modify if needed to reproduce the problem. Then send me back the modified version with clear reproduction steps. Once we've reached that point, I'll take another look.

[SwiftMessagesTest-477.zip](https://github.com/SwiftKickMobile/SwiftMessages/files/7041161/SwiftMessagesTest-477.zip)
